src/main/java/com/webfleet/oauth/Application.java

```diff
@@ -0,0 +1,23 @@
+package com.webfleet.oauth;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+
+@SpringBootApplication
+public class Application extends SpringBootServletInitializer
+{
```

Comment:
Not sure if we should expose our somewhat special code style to the public :)

Reply:
Finally opted for using a more "common" style (default IntelliJ java style) and exported it using `.editorconfig` format (https://editorconfig.org/)